docs/drilldowns/drilldowns.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[drilldowns]]
+== Use drilldowns for dashboard actions
+
+Drilldowns, also known as custom actions, allow you to configure a
+workflow for analyzing and troubleshooting your data.
+Using a drilldown, you can navigate from one dashboard to another,
+taking the current time range, filters, and other parameters with you,
+so the context remains the same. You can continue your analysis from a new perspective.
+
+For example, you might have a dashboard that shows the overall status of multiple data centers.
+You can create a drilldown that navigates from this dashboard to a dashboard
+that shows a single data center or server.
+
+[float]
+[[how-drilldowns-work]]
+=== How drilldowns work
+
+Drilldowns are {kib} actions that you configure and store
```

Comment:
One very minor nitpicky thing: is it worth explaining the functionality to the end user in terms of actions and saved objects?  I’m not sure those terms necessarily mean anything to the end users.


Maybe change it to something like "Drilldowns are user-configurable actions that are stored with the dashboard metadata”?
I don’t have a strong opinion one way or another - just suggesting an alternative.